.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - name: Setup nightly Rust
-      run: rustup toolchain install nightly-2022-12-12 --profile minimal --target wasm32-unknown-unknown --component rust-src
+      run: rustup toolchain install nightly-2024-08-02 --profile minimal --target wasm32-unknown-unknown --component rust-src
```

Comment:
These shouldn't be necessary with `rust-toolchain.toml` now, right?

Reply:
probably right, I just blindly replaced the toolchain here.

Gonna test it.